cpu/stm32f1/lpm_arch.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 INRIA
- * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
@DipSwitch seems preoccupied. I guess we can settle this at a later point if need be.